documentation/puppet_server_metrics_performance.markdown

```diff
@@ -57,7 +57,7 @@ As a general rule, adding a JRuby requires a bare minimum of 40MB of memory unde
 
 For real-world catalogs, you can generally add an absolute minimum of 15MB for each additional JRuby. We calculated this amount by comparing a minimal catalog compilation to compiling a catalog for a [basic role](https://github.com/puppetlabs/puppetlabs-puppetserver_perf_control/blob/production/site/role/manifests/by_size/small.pp) that installs Tomcat and Postgres servers.
 
-Your Puppet-managed infrastructure is probably larger and more complex than that test scenario, and every complication adds more to each additional JRuby's memory requirements. (For instance, we recommend assuming that Puppet Server will use [at least 512MB per JRuby](https://docs.puppet.com/pe/latest/config_puppetserver.html) while under load.) You can calculate a similar value unique to your infrastructure by measuring `puppetserver` memory usage during your infrastructure's catalog compilations and comparing it to compiling a minimal catalog for a similar number of nodes.
+Your Puppet-managed infrastructure is probably larger and more complex than that test scenario, and every complication adds more to each additional JRuby's memory requirements. (For instance, we recommend assuming that Puppet Server will use [at least 512MB per JRuby](https://puppet.com/docs/pe/2017.3/configuring/config_puppetserver.html) while under load.) You can calculate a similar value unique to your infrastructure by measuring `puppetserver` memory usage during your infrastructure's catalog compilations and comparing it to compiling a minimal catalog for a similar number of nodes.
```

Comment:
I think this should point to:
https://puppet.com/docs/pe/latest/configuring/config_puppetserver.html
rather than
https://puppet.com/docs/pe/2017.3/configuring/config_puppetserver.html